format-specs/spatial-type-guidance.md

```diff
@@ -0,0 +1,328 @@
+# Spatial Type Implementation Review
+
+The GeoParquet specification is an example of a **Spatial Type** implementation
+that attempts to combine principles from both well established existing geospatial libraries
+and cutting edge data engineering components like [Apache Parquet](https://parquet.apache.org),
+[Apache Arrow](https://arrow.apache.org), [Apache Iceberg](https://iceberg.apache.org).
+These components are used by major data warehouse vendors like
+[BigQuery](https://cloud.google.com/bigquery),
+[Snowflake](https://snowflake.com), [Databricks](https://databricks.com). Broadly,
+GeoParquet was
+
+## Spatial Types?
+
+By **Spatial Type**, we are referring to a library, specification, or file format
+that intends to represent points, lines, polygons, or combinations thereof and relate
+them to a position on the surface of the Earth[^1]. We will also constrain this
+definition to only include attempts to *formalize* the representation of this
+data within an existing type system or format that is otherwise *not* a dedicated
+spatial type. Essentially, if you are storing data in a non-spatial format that you
+intend to be put on a map, and you expect somebody else to interpret it in the same
+way that you did, that data comes from a spatial type implementation.
+
+[^1]: Technically this also applies if you are on the surface of another planet or
+celestial body, and also applies if you are above or below the surface. We'll
+discuss this later.
+
+## If you don't read anything else
+
+The simplest possible spatial type implementation that can interoperate with most existing
+Spatial Type implementations without loosing information is a
+"Geometry" [and/or "Geography"](#geometry-and-geography) type, parameterized with `crs`
+([Coordinate Reference System](#coordinate-reference-systems)) as a string, with values
+encoded as [Well-known binary](#well-known-binary) (if your format has a native way
+to store an array of bytes) or [Well-known text](#well-known-text) (otherwise).
+
+In general, the motivation behind the suggestions in this document are to ensure that
+new Spatial Type implementations:
+
+- **Capture producer intent**: Users of your Spatial Type should not have to discard
+  information when converting spatial data from elsewhere. That means
+  if your Spatial Type intends to interoperate with any other Spatial Type, you need
+  to provide a mechanism to locate a full Coordinate Reference System definition.
+  For example, [GeoPandas](https://geopandas.org) stores a coordinate reference system
+  at the type level of each array of geometries.
```

Comment:
```suggestion
  for each column of geometries.
```

(type level is perhaps an implementation detail that makes this just a little harder to follow here as a general example)